docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '2'
+
+services:
+  zookeeper:
+    image: zookeeper
+    hostname: zookeeper
+    ports:
+      - "2181:2181"
+  bookkeeper:
+    image: apache/bookkeeper
+    build: .
+    depends_on:
+      - zookeeper
```

Comment:
we may need more work on 'depends_on' here, because it is a simple deploy order, but not wait for the service to be ready, please reference https://docs.docker.com/compose/startup-order/ for more details.